compiler-core/src/analyse.rs

```diff
@@ -126,6 +127,7 @@ pub fn infer_module<A>(
     target_support: TargetSupport,
     line_numbers: LineNumbers,
     src_path: Utf8PathBuf,
+    config: &PackageConfig,
```

Comment:
This is now duplicating the `package` parameter above. Let's remove that and take it from this config